librz/type/path.c

```diff
@@ -68,7 +68,10 @@ static RzType *path_walker_parse_bracket(const RzTypeDB *typedb, RzType *parent,
 			return NULL;
 		}
 
-		size_t idx = strtoull(strndup(&path[tok_beg], *i - tok_beg), NULL, 10);
+		char *idx_str = strndup(&path[tok_beg], *i - tok_beg);
+		size_t idx = strtoull(idx_str, NULL, 10);
```

Comment:
use `rz_str_ndup()` and `rz_num_math(NULL, idx_str)`

Reply:
fixed, thanks